src/foam/u2/view/ChoiceView.js

```diff
@@ -244,8 +244,9 @@ foam.CLASS({
               .enableClass('selection-made', self.index$.map((index) => index !== -1))
             .end();
         }
-
-        return self.E().translate(self.text + ".name", self.text);
+        return self.text ?
```

Comment:
I don't think this is correct. You're relying on timing, but text$ might not originally be empty. How about we do something like:
self.E().add(self.text$.map(t => self.translationService.translate(...)));   ?

Reply:
@kgrgreer if `text` is a part of the slot at line 229 then would we still have timing issue?